projects/igniteui-angular/src/lib/core/styles/components/drop-down/_drop-down-theme.scss

```diff
@@ -312,6 +312,7 @@
         color: --var($theme, 'focused-item-text-color');
 
         &:hover {
+            color: --var($theme, 'focused-item-text-color');
```

Comment:
Isn't this exactly the same as the declaration above? I couldn't notice any difference with this line removed.

Reply:
In order to see the problem focus one of the menu items and then hover over it. The problem is that we don't remove the initial class igx-drop-down__item but instead we add a new one igx-drop-down__item--focus. The first class preserves his hover style and the text turns black.